vertx-pg-client/src/main/java/io/vertx/pgclient/impl/codec/PgEncoder.java

```diff
@@ -193,16 +193,10 @@ void writeStartupMessage(StartupMessage msg) {
     Util.writeCStringUTF8(out, msg.username);
     writeCString(out, StartupMessage.BUFF_DATABASE);
     Util.writeCStringUTF8(out, msg.database);
-    writeCString(out, StartupMessage.BUFF_APPLICATION_NAME);
-    writeCString(out, StartupMessage.BUFF_VERTX_PG_CLIENT);
-    writeCString(out, StartupMessage.BUFF_CLIENT_ENCODING);
-    writeCString(out, StartupMessage.BUFF_UTF8);
-    writeCString(out, StartupMessage.BUFF_DATE_STYLE);
-    writeCString(out, StartupMessage.BUFF_ISO);
-    writeCString(out, StartupMessage.BUFF_INTERVAL_STYLE);
-    writeCString(out, StartupMessage.BUFF_INTERVAL_STYLE_TYPE);
-    writeCString(out, StartupMessage.BUFF_EXTRA_FLOAT_DIGITS);
-    writeCString(out, StartupMessage.BUFF_2);
+    msg.properties.forEach((key, value) -> {
```

Comment:
rather use a for-loop on `properties.entries()` to avoid capturing lambda

Reply:
updated in https://github.com/eclipse-vertx/vertx-sql-client/pull/344/commits/2a068f7acc9229396a6eff37d1d38beb87820757.